thrift/lib/cpp2/protocol/test/BinaryProtocolTest.cpp

```diff
@@ -56,7 +56,9 @@ TEST_F(BinaryProtocolTest, writeInvalidBool) {
         auto s = std::string();
         q.appendToString(s);
         // Die on success.
-        CHECK(s != std::string(1, '\0')) << "invalid bool value";
+        if(s != std::string(1, '\0')) {
+          exit(1);
+        }
```

Comment:
Why change this?

Reply:
> There seems to be some incompatibility between gtest and glog causing CHECK(..) calls inside EXPECTED_DEATH to fail with the error 'macro "EXPECT_EXIT" passed 4 arguments, but takes just 3'. I rewrote the failing lines to not use glog for now.

I was unable to resolve this issue myself, possible an incompatibility between gtest and glog versions. That's why I replaced the `CHECK(..)` call with a manual check for now. Any idea why the previous call is failing here?

Reply:
Please move it to a separate PR since it's unrelated to build configs.